gcsweb/README.md

```diff
@@ -24,3 +24,11 @@ links to specific files.
 2. `gcsweb` is not designed for initial browsing (yet?) - it doesn't list
 which GCS buckets are available, and because bucket is a part of URL, you
 need a documented link to browse.
+
+#### Installation and deployment
+
+`gcsweb` is built from source code at
+http://github.com/kubernetes/test-infra/gcsweb into Docker container, which
```

Comment:
That is not a valid URL - it was a go-path in email.  sorry.  https://github.com/kubernetes/test-infra/tree/master/gcsweb